server/graphql_schemas/tests.py

```diff
@@ -0,0 +1,148 @@
+import datetime
+from contextlib import suppress
+from django.utils import timezone
+import pytz
+
+from django.test import RequestFactory
+from django.contrib.auth.models import User
+from snapshottest.django import TestCase
+
+from graphene.test import Client
+from graphql import GraphQLError
+
+from .schema import schema
+from api.models import Event, GoogleUser, Category
+
+timezone.now()
+date = datetime.datetime(2018, 11, 20, 20, 8, 7, 127325, tzinfo=pytz.UTC)
+client = Client(schema)
+request_factory = RequestFactory()
+request = request_factory.get('/graphql')
+
+
+def create_user(googleId):
+    user = User.objects.create_user(
+            username='test_user{}'.format(googleId),
+            password='fakepassword',
+            email='testemail{}@email.com'.format(googleId)
+            )
+
+    google_user = GoogleUser(
+            google_id=googleId,
+            app_user=user
+            )
+    google_user.save()
+    return google_user
+
+
+def create_admin_user(googleId):
+    user = User.objects.create_superuser(
+            username='test_user{}'.format(googleId),
+            password='fakepassword',
+            email='testemail{}@email.com'.format(googleId)
+            )
+
+    google_user = GoogleUser(
+            google_id=googleId,
+            app_user=user
+            )
+    google_user.save()
+    return google_user
+
+
+def create_event(event_creator, category, active=True, id=1):
+    event = Event(
+                id=id,
+                title="test title",
+                description="test description",
+                venue="test venue",
+                time="3PM",
+                date=date,
+                creator=event_creator,
+                social_event=category,
+                active=active,
+                created_at=date
+            )
+    event.save()
+    return event
+
+
+def create_category():
+    category = Category(
+                    id=1,
+                    name="social event",
+                    featured_image="featured_image",
+                    description="a test description"
+            )
+    return category
+
+
+class EventTestCase(TestCase):
+    """
+    Tests the events api queries and mutations
+    """
+
+    def setUp(self):
+        self.event_creator = create_user("creatorId")
+        self.non_event_creator = create_user("ncreatorId")
+        self.admin = create_admin_user("adminId")
+        self.category = create_category()
+        create_event(self.event_creator, self.category)
+
+    def tearDown(self):
+        Event.objects.all().delete()
+        GoogleUser.objects.all().delete()
```

Comment:
Category should be torn down too after the test.

Reply:
@makindemayowa. It doesn't really save to the database, just creates an instance of it. I didn't think there would be any need to tear it down.